tests/autest.sh

```diff
@@ -29,7 +29,7 @@ if [ ! -f ./env-test/bin/autest ]; then\
 # this is for rhel or centos systems
 test -r /opt/rh/rh-python35/enable && . /opt/rh/rh-python35/enable
 . env-test/bin/activate
-./env-test/bin/autest -D gold_tests $*
+./env-test/bin/autest -D gold_tests "$@"
```

Comment:
The change here is a small correction to the way we are passing arguments. Using `$*` is almost never what we want. Using `"$@"` will properly quote individual arguments without the shell re-evaluating them.

To answer the question, the env-test directory is generated when running autest.sh via a "bootstrap" procedure.

This isn't a change that is strictly necessary, so I could remove it. I just thought it should be fixed in case someone tries arguments with spaces, for example.